tools/virtualization/installing-qemu-on-fedora-linux.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Installing QEMU on Fedora Linux  
+subsection: virtualization  
+order: 1  
+---
+
+# Installing QEMU on Fedora Linux
```

Comment:
Everything here on Fedora Developer Portal is on Fedora Linux. No need to mention it :). Also I'm quite missing the description and some basic usage or reference to a guide.

```suggestion
# Installing QEMU
```

Reply:
Addressed here https://github.com/developer-portal/content/pull/394/commits/c2a00be8cf04a4bb8184e9ca31177175e4d631ec